internal/pkg/deploy/cloudformation/cloudformation_integration_test.go

```diff
@@ -1,4 +1,4 @@
-// +build integration
+// build integration
```

Comment:
huh what's the difference? why did we remove it

Reply:
Whoops. That's why the tests were failing!